src/pages/index.js

```diff
@@ -159,21 +253,27 @@ cardModalBtn.addEventListener("click", () => {
   openModal(cardModal);
 });
 
+cancelBtn.addEventListener("click", () => {
+  closeModal();
+});
+
 closeBtns.forEach((button) => {
   const modal = button.closest(".modal");
```

Comment:
`  const modal = button.closest(".modal");
`

It's not needed anymore because `closeModal` doesn't accept arguments

```js
closeBtns.forEach((button) => {
  button.addEventListener("click", closeModal);
});
```